### ISSUE TITLE
Errors after installation

### ISSUE THREAD

**Issue by jerry1970** [madflow/flow-netbeans-markdown#71]
Just found out about this plugin. Sounds amazing but I get the following errors. My NetBeans version: DEV 201502020002 (64 bits) (that's a development version after 8.0.2). I guess it's a conflict with another plugin but have no clue on how to go at this. Any ideas? Thanks!

I'm going to install the official 8.0.2 and try again...

```
java.lang.IllegalStateException
    at org.objectweb.asm.tree.ClassNode.<init>(Unknown Source)
    at org.parboiled.transform.ParserClassNode.<init>(ParserClassNode.java:41)
    at org.parboiled.transform.ParserTransformer.extendParserClass(ParserTransformer.java:43)
    at org.parboiled.transform.ParserTransformer.transformParser(ParserTransformer.java:38)
    at org.parboiled.Parboiled.createParser(Parboiled.java:54)
Caused: java.lang.RuntimeException: Error creating extended parser class: null
    at org.parboiled.Parboiled.createParser(Parboiled.java:58)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:92)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:61)
    at flow.netbeans.markdown.csl.MarkdownParser.parse(MarkdownParser.java:33)
    at org.netbeans.modules.parsing.impl.TaskProcessor.callParse(TaskProcessor.java:621)
    at org.netbeans.modules.parsing.impl.SourceCache.getResult(SourceCache.java:251)
    at org.netbeans.modules.parsing.api.ResultIterator.getParserResult(ResultIterator.java:138)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work$2.run(RepositoryUpdater.java:3168)
    at org.netbeans.modules.parsing.impl.TaskProcessor.callUserTask(TaskProcessor.java:609)
    at org.netbeans.modules.parsing.api.ParserManager$UserTaskAction.run(ParserManager.java:153)
    at org.netbeans.modules.parsing.api.ParserManager$UserTaskAction.run(ParserManager.java:137)
    at org.netbeans.modules.parsing.impl.TaskProcessor$2.call(TaskProcessor.java:204)
    at org.netbeans.modules.parsing.impl.TaskProcessor$2.call(TaskProcessor.java:201)
    at org.netbeans.modules.masterfs.filebasedfs.utils.FileChangedManager.priorityIO(FileChangedManager.java:176)
    at org.netbeans.modules.masterfs.providers.ProvidedExtensions.priorityIO(ProvidedExtensions.java:360)
    at org.netbeans.modules.parsing.nb.DataObjectEnvFactory.runPriorityIO(DataObjectEnvFactory.java:141)
    at org.netbeans.modules.parsing.impl.Utilities.runPriorityIO(Utilities.java:88)
    at org.netbeans.modules.parsing.impl.TaskProcessor.runUserTask(TaskProcessor.java:201)
Caused: org.netbeans.modules.parsing.spi.ParseException
    at org.netbeans.modules.parsing.impl.TaskProcessor.runUserTask(TaskProcessor.java:209)
    at org.netbeans.modules.parsing.api.ParserManager.parse(ParserManager.java:104)
[catch] at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work.indexEmbedding(RepositoryUpdater.java:3140)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work.doIndex(RepositoryUpdater.java:2861)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work.access$400(RepositoryUpdater.java:2153)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work$1.run(RepositoryUpdater.java:2635)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work$1.run(RepositoryUpdater.java:2633)
    at org.netbeans.modules.parsing.impl.indexing.errors.TaskCache.refreshTransaction(TaskCache.java:546)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work.index(RepositoryUpdater.java:2633)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work$3.call(RepositoryUpdater.java:3286)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work$3.call(RepositoryUpdater.java:3252)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$3.run(RepositoryUpdater.java:2126)
    at org.openide.util.lookup.Lookups.executeWith(Lookups.java:295)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater.runInContext(RepositoryUpdater.java:2122)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater.runInContext(RepositoryUpdater.java:2103)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater.access$1100(RepositoryUpdater.java:156)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work.scanFiles(RepositoryUpdater.java:3252)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$FileListWork.getDone(RepositoryUpdater.java:3771)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Work.doTheWork(RepositoryUpdater.java:3406)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Task._run(RepositoryUpdater.java:6170)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Task.access$4300(RepositoryUpdater.java:5817)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Task$3$1.run(RepositoryUpdater.java:6088)
    at org.openide.util.lookup.Lookups.executeWith(Lookups.java:295)
    at org.netbeans.modules.parsing.impl.RunWhenScanFinishedSupport.performScan(RunWhenScanFinishedSupport.java:106)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Task$3.call(RepositoryUpdater.java:6084)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Task$3.call(RepositoryUpdater.java:6080)
    at org.netbeans.modules.masterfs.filebasedfs.utils.FileChangedManager.priorityIO(FileChangedManager.java:176)
    at org.netbeans.modules.masterfs.providers.ProvidedExtensions.priorityIO(ProvidedExtensions.java:360)
    at org.netbeans.modules.parsing.nb.DataObjectEnvFactory.runPriorityIO(DataObjectEnvFactory.java:141)
    at org.netbeans.modules.parsing.impl.Utilities.runPriorityIO(Utilities.java:88)
    at org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater$Task.run(RepositoryUpdater.java:6080)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1425)
    at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:59)
    at org.openide.util.lookup.Lookups.executeWith(Lookups.java:294)
    at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2038)

java.lang.IllegalStateException
    at org.objectweb.asm.tree.ClassNode.<init>(Unknown Source)
    at org.parboiled.transform.ParserClassNode.<init>(ParserClassNode.java:41)
    at org.parboiled.transform.ParserTransformer.extendParserClass(ParserTransformer.java:43)
    at org.parboiled.transform.ParserTransformer.transformParser(ParserTransformer.java:38)
    at org.parboiled.Parboiled.createParser(Parboiled.java:54)
Caused: java.lang.RuntimeException: Error creating extended parser class: null
    at org.parboiled.Parboiled.createParser(Parboiled.java:58)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:92)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:61)
    at flow.netbeans.markdown.highlighter.MarkdownLexer.tokenizeInput(MarkdownLexer.java:73)
    at flow.netbeans.markdown.highlighter.MarkdownLexer.nextToken(MarkdownLexer.java:40)
    at org.netbeans.lib.lexer.LexerInputOperation.nextToken(LexerInputOperation.java:215)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenOrEmbeddingImpl(IncTokenList.java:195)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenOrEmbedding(IncTokenList.java:190)
    at org.netbeans.lib.lexer.LexerUtilsConstants.tokenIndexLazyTokenCreation(LexerUtilsConstants.java:317)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenIndex(IncTokenList.java:172)
    at org.netbeans.api.lexer.TokenSequence.move(TokenSequence.java:680)
    at org.netbeans.modules.editor.lib2.highlighting.SyntaxHighlighting$HSImpl.<init>(SyntaxHighlighting.java:369)
    at org.netbeans.modules.editor.lib2.highlighting.SyntaxHighlighting.getHighlights(SyntaxHighlighting.java:139)
    at org.netbeans.modules.editor.lib2.highlighting.DirectMergeContainer$HlSequence.<init>(DirectMergeContainer.java:197)
    at org.netbeans.modules.editor.lib2.highlighting.DirectMergeContainer.getHighlights(DirectMergeContainer.java:104)
    at org.netbeans.modules.editor.lib2.highlighting.HighlightsReader.<init>(HighlightsReader.java:62)
    at org.netbeans.modules.editor.lib2.view.HighlightsViewFactory.restart(HighlightsViewFactory.java:201)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder$FactoryState.init(ViewBuilder.java:1324)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder.createViews(ViewBuilder.java:724)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder.createReplaceRepaintViews(ViewBuilder.java:685)
    at org.netbeans.modules.editor.lib2.view.ViewUpdates.reinitAllViews(ViewUpdates.java:206)
    at org.netbeans.modules.editor.lib2.view.DocumentViewOp.checkViewsInited(DocumentViewOp.java:634)
    at org.netbeans.modules.editor.lib2.view.DocumentView.getPreferredSpan(DocumentView.java:251)
    at javax.swing.plaf.basic.BasicTextUI$RootView.getPreferredSpan(BasicTextUI.java:1353)
    at javax.swing.plaf.basic.BasicTextUI.getPreferredSize(BasicTextUI.java:919)
    at javax.swing.JComponent.getPreferredSize(JComponent.java:1660)
    at javax.swing.JEditorPane.getPreferredSize(JEditorPane.java:1332)
    at javax.swing.ScrollPaneLayout.layoutContainer(ScrollPaneLayout.java:791)
    at java.awt.Container.layout(Container.java:1508)
    at java.awt.Container.doLayout(Container.java:1497)
    at java.awt.Container.validateTree(Container.java:1693)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validate(Container.java:1628)
    at org.netbeans.core.windows.view.ui.DesktopImpl$LayeredLayout.layoutContainer(DesktopImpl.java:554)
    at java.awt.Container.layout(Container.java:1508)
    at java.awt.Container.doLayout(Container.java:1497)
    at java.awt.Container.validateTree(Container.java:1693)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validate(Container.java:1628)
    at javax.swing.RepaintManager$3.run(RepaintManager.java:704)
    at javax.swing.RepaintManager$3.run(RepaintManager.java:702)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at javax.swing.RepaintManager.validateInvalidComponents(RepaintManager.java:701)
    at javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1719)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:749)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:702)
    at java.awt.EventQueue$3.run(EventQueue.java:696)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:719)
    at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
[catch] at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

java.lang.IllegalStateException
    at org.objectweb.asm.tree.ClassNode.<init>(Unknown Source)
    at org.parboiled.transform.ParserClassNode.<init>(ParserClassNode.java:41)
    at org.parboiled.transform.ParserTransformer.extendParserClass(ParserTransformer.java:43)
    at org.parboiled.transform.ParserTransformer.transformParser(ParserTransformer.java:38)
    at org.parboiled.Parboiled.createParser(Parboiled.java:54)
Caused: java.lang.RuntimeException: Error creating extended parser class: null
    at org.parboiled.Parboiled.createParser(Parboiled.java:58)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:92)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:61)
    at flow.netbeans.markdown.highlighter.MarkdownLexer.tokenizeInput(MarkdownLexer.java:73)
    at flow.netbeans.markdown.highlighter.MarkdownLexer.nextToken(MarkdownLexer.java:40)
    at org.netbeans.lib.lexer.LexerInputOperation.nextToken(LexerInputOperation.java:215)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenOrEmbeddingImpl(IncTokenList.java:195)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenOrEmbedding(IncTokenList.java:190)
    at org.netbeans.lib.lexer.LexerUtilsConstants.tokenIndexLazyTokenCreation(LexerUtilsConstants.java:317)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenIndex(IncTokenList.java:172)
    at org.netbeans.api.lexer.TokenSequence.move(TokenSequence.java:680)
    at org.netbeans.modules.editor.lib2.highlighting.SyntaxHighlighting$HSImpl.<init>(SyntaxHighlighting.java:369)
    at org.netbeans.modules.editor.lib2.highlighting.SyntaxHighlighting.getHighlights(SyntaxHighlighting.java:139)
    at org.netbeans.modules.editor.lib2.highlighting.DirectMergeContainer$HlSequence.<init>(DirectMergeContainer.java:197)
    at org.netbeans.modules.editor.lib2.highlighting.DirectMergeContainer.getHighlights(DirectMergeContainer.java:104)
    at org.netbeans.modules.editor.lib2.highlighting.HighlightsReader.<init>(HighlightsReader.java:62)
    at org.netbeans.modules.editor.lib2.view.HighlightsViewFactory.restart(HighlightsViewFactory.java:201)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder$FactoryState.init(ViewBuilder.java:1324)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder.createViews(ViewBuilder.java:724)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder.createReplaceRepaintViews(ViewBuilder.java:685)
    at org.netbeans.modules.editor.lib2.view.ViewUpdates.reinitAllViews(ViewUpdates.java:206)
    at org.netbeans.modules.editor.lib2.view.DocumentViewOp.checkViewsInited(DocumentViewOp.java:634)
    at org.netbeans.modules.editor.lib2.view.DocumentView.getPreferredSpan(DocumentView.java:251)
    at javax.swing.plaf.basic.BasicTextUI$RootView.getPreferredSpan(BasicTextUI.java:1353)
    at javax.swing.plaf.basic.BasicTextUI.getPreferredSize(BasicTextUI.java:919)
    at javax.swing.JComponent.getPreferredSize(JComponent.java:1660)
    at javax.swing.JEditorPane.getPreferredSize(JEditorPane.java:1332)
    at javax.swing.ScrollPaneLayout.layoutContainer(ScrollPaneLayout.java:791)
    at java.awt.Container.layout(Container.java:1508)
    at java.awt.Container.doLayout(Container.java:1497)
    at java.awt.Container.validateTree(Container.java:1693)
    at java.awt.Container.validate(Container.java:1628)
    at javax.swing.RepaintManager$3.run(RepaintManager.java:704)
    at javax.swing.RepaintManager$3.run(RepaintManager.java:702)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at javax.swing.RepaintManager.validateInvalidComponents(RepaintManager.java:701)
    at javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1719)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:749)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:702)
    at java.awt.EventQueue$3.run(EventQueue.java:696)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:719)
    at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
[catch] at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

java.lang.IllegalStateException
    at org.objectweb.asm.tree.ClassNode.<init>(Unknown Source)
    at org.parboiled.transform.ParserClassNode.<init>(ParserClassNode.java:41)
    at org.parboiled.transform.ParserTransformer.extendParserClass(ParserTransformer.java:43)
    at org.parboiled.transform.ParserTransformer.transformParser(ParserTransformer.java:38)
    at org.parboiled.Parboiled.createParser(Parboiled.java:54)
Caused: java.lang.RuntimeException: Error creating extended parser class: null
    at org.parboiled.Parboiled.createParser(Parboiled.java:58)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:92)
    at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:61)
    at flow.netbeans.markdown.highlighter.MarkdownLexer.tokenizeInput(MarkdownLexer.java:73)
    at flow.netbeans.markdown.highlighter.MarkdownLexer.nextToken(MarkdownLexer.java:40)
    at org.netbeans.lib.lexer.LexerInputOperation.nextToken(LexerInputOperation.java:215)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenOrEmbeddingImpl(IncTokenList.java:195)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenOrEmbedding(IncTokenList.java:190)
    at org.netbeans.lib.lexer.LexerUtilsConstants.tokenIndexLazyTokenCreation(LexerUtilsConstants.java:317)
    at org.netbeans.lib.lexer.inc.IncTokenList.tokenIndex(IncTokenList.java:172)
    at org.netbeans.api.lexer.TokenSequence.move(TokenSequence.java:680)
    at org.netbeans.modules.editor.lib2.highlighting.SyntaxHighlighting$HSImpl.<init>(SyntaxHighlighting.java:369)
    at org.netbeans.modules.editor.lib2.highlighting.SyntaxHighlighting.getHighlights(SyntaxHighlighting.java:139)
    at org.netbeans.modules.editor.lib2.highlighting.DirectMergeContainer$HlSequence.<init>(DirectMergeContainer.java:197)
    at org.netbeans.modules.editor.lib2.highlighting.DirectMergeContainer.getHighlights(DirectMergeContainer.java:104)
    at org.netbeans.modules.editor.lib2.highlighting.HighlightsReader.<init>(HighlightsReader.java:62)
    at org.netbeans.modules.editor.lib2.view.HighlightsViewFactory.restart(HighlightsViewFactory.java:201)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder$FactoryState.init(ViewBuilder.java:1324)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder.createViews(ViewBuilder.java:724)
    at org.netbeans.modules.editor.lib2.view.ViewBuilder.createReplaceRepaintViews(ViewBuilder.java:685)
    at org.netbeans.modules.editor.lib2.view.ViewUpdates.reinitAllViews(ViewUpdates.java:206)
    at org.netbeans.modules.editor.lib2.view.DocumentViewOp.checkViewsInited(DocumentViewOp.java:634)
    at org.netbeans.modules.editor.lib2.view.DocumentView.getPreferredSpan(DocumentView.java:251)
    at javax.swing.plaf.basic.BasicTextUI$RootView.getPreferredSpan(BasicTextUI.java:1353)
    at javax.swing.plaf.basic.BasicTextUI.getPreferredSize(BasicTextUI.java:919)
    at javax.swing.JComponent.getPreferredSize(JComponent.java:1660)
    at javax.swing.JEditorPane.getPreferredSize(JEditorPane.java:1332)
    at javax.swing.ScrollPaneLayout.layoutContainer(ScrollPaneLayout.java:791)
    at java.awt.Container.layout(Container.java:1508)
    at java.awt.Container.doLayout(Container.java:1497)
    at java.awt.Container.validateTree(Container.java:1693)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validate(Container.java:1628)
    at org.netbeans.core.windows.view.ui.DesktopImpl$LayeredLayout.layoutContainer(DesktopImpl.java:554)
    at java.awt.Container.layout(Container.java:1508)
    at java.awt.Container.doLayout(Container.java:1497)
    at java.awt.Container.validateTree(Container.java:1693)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validateTree(Container.java:1702)
    at java.awt.Container.validate(Container.java:1628)
    at javax.swing.RepaintManager$3.run(RepaintManager.java:704)
    at javax.swing.RepaintManager$3.run(RepaintManager.java:702)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at javax.swing.RepaintManager.validateInvalidComponents(RepaintManager.java:701)
    at javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1719)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:749)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:702)
    at java.awt.EventQueue$3.run(EventQueue.java:696)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:719)
    at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
[catch] at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by rmburkhead** [comment]
I loaded the source for this plugin into a later DEV build of the IDE (201503100001) and noticed that there were a few errors where some platform/IDE APIs appear to have changed. The dev version of NetBeans is now on track for a version 8.1 release, so the APIs are probably in flux and not compatible with the 8.0 libraries used for this plugin. (I did not try to install the plugin, just did a superficial scan of the source for errors.)

The plugin does install fine on version 8.0.2 of the IDE (I'm using build 201411181905).

**Comment by jerry1970** [comment]
I installed NB8.0.2 on my Win8 machine and the MD plugin works fine! :)

I remember that I started using the NB DEV versions when I was on a unix machine and there were errors in NB when the window was maximised on anything but the primary monitor. I am on Win now (Win8 at the office, Win7 at home) where that error does not apply so no reason not to go back to the 8.0.2...

Thanks for the reply! I hope you will be able and have time to update the plugin for NB 8.1 when that is released. Cheers!
